src/Nethermind/Nethermind.DataMarketplace.Consumers/Refunds/Services/RefundClaimant.cs

```diff
@@ -57,6 +57,11 @@ public async Task<RefundClaimStatus> TryClaimRefundAsync(DepositDetails deposit,
             ulong now = _timestamper.UnixTime.Seconds;
             if (!deposit.CanClaimRefund(now))
             {
+                var timeLeftToClaimRefund = deposit.GetTimeLeftToClaimRefund(now);
+                if (timeLeftToClaimRefund > 0)
+                {
+                    if (_logger.IsError) _logger.Info($"Time left to claim a refund: {timeLeftToClaimRefund} seconds.");
```

Comment:
Not sure with how many seconds on average are we dealing with but maybe it's better to have hh:mm:ss format in logs - you can make it with `TimeSpan.FromSeconds(seconds).ToString()`